src/meep.hpp

```diff
@@ -2077,13 +2077,13 @@ class fields {
   /* of DFT fields, and  evaluating overlap integrals     */
   /* flux and mode fields.)                               */
   /********************************************************/
-  std::complex<double> process_dft_component(dft_chunk **chunklists, int num_chunklists,
-                                             int num_freq, component c, const char *HDF5FileName,
-                                             std::complex<double> **field_array = 0, int *rank = 0,
-                                             size_t *dims = 0, direction *dirs = 0,
-                                             void *mode1_data = 0, void *mode2_data = 0,
-                                             component c_conjugate = Ex, bool *first_component = 0,
-                                             bool retain_interp_weights = true);
+  std::complex<realnum> process_dft_component(dft_chunk **chunklists, int num_chunklists,
```

Comment:
Why did the return value change?

Reply:
Return type reverted to `std::complex<double>`.